packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -3,10 +3,12 @@
 
 :local {
   .nav-container {
+    background: inherit;
```

Comment:
I have noticed that also. This happens if the background is a gradient. Tested with a background color and in that case, profile gets same as the background. The mobile nav background is one color, in that case, it is working as expected. The only way I could get the profile to have the same color as the gradient background was making it transparent. But in that case, it causes an issue when there are more nav elements and scroll is present. I am trying to find a solution. Please let me know if you have a solution in mind that I can try. Thank you.